androidApp/src/main/java/com/mbta/tid/mbta_app/android/component/UpcomingTripView.kt

```diff
@@ -313,7 +316,17 @@ fun UpcomingTripView(
                         textAlign = TextAlign.End
                     )
             }
-        is UpcomingTripViewState.Loading -> CircularProgressIndicator(modifier)
+        is UpcomingTripViewState.Loading ->
+            CompositionLocalProvider(IsLoadingSheetContents provides true) {
```

Comment:
suggestion (non-blocking): This could use the `import com.eygraber.compose.placeholder.material3.placeholder` modifier directly rather than having to force `IsLoadingSheetContents` state.

Reply:
If we ever decide we want our placeholders to be styled in some more specific way, leaving it like this will mean we don't have to come back and change this specific one separately.